pkg/cloud/azure/azurestorage.go

```diff
@@ -142,6 +131,7 @@ func (c StorageClient) UploadPageBlob(metadata BlobMetadata, fileName string, th
 	// Run the upload
 	run := true
 	var wg sync.WaitGroup
+	zeros := bytes.Repeat([]byte{0}, PageBlobMaxUploadPagesBytes)
```

Comment:
Nitpick: this could be simply `zeros := make([]byte, PageBlobMaxUploadPagesBytes)`